src/components/landing/UpcomingAnnouncements.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+
+import { useState, useEffect } from "react";
+import HourglassIcon from "@/icons/HourglassIcon";
+import AnnouncementBanner from "../shared/AnnouncementBanner";
+import { Announcement } from "@/models/Announcement";
+
+interface Props {
+  announcements: Announcement[];
+}
+
+export default function UpcomingAnnouncements({ announcements }: Props) {
+  const futureAnnouncements = announcements
+    .filter((announcement) => new Date(announcement.startDate) > new Date())
+    .sort(
+      (a, b) =>
+        new Date(a.startDate).getTime() - new Date(b.startDate).getTime()
+    );
+  const [timeRemaining, setTimeRemaining] = useState({
+    days: 0,
+    hours: 0,
+    minutes: 0,
+    seconds: 0,
+  });
+
+  const calculateTimeRemaining = (startDate: Date) => {
+    const now = new Date();
+    const timeDiff = startDate.getTime() - now.getTime();
+    if (timeDiff <= 0) return { days: 0, hours: 0, minutes: 0, seconds: 0 };
+
+    const seconds = Math.floor(timeDiff / 1000);
+    const minutes = Math.floor(seconds / 60);
+    const hours = Math.floor(minutes / 60);
+    const days = Math.floor(hours / 24);
+
+    return {
+      days: Math.min(days, 99),
+      hours: hours % 24,
+      minutes: minutes % 60,
+      seconds: seconds % 60,
+    };
+  };
+
+  useEffect(() => {
+    if (futureAnnouncements.length === 0) return;
+
+    const firstFutureAnnouncement = futureAnnouncements[0];
+
+    const startDate = new Date(firstFutureAnnouncement.startDate);
+
+    const updateCountdown = () => {
+      setTimeRemaining(calculateTimeRemaining(startDate));
+    };
+
+    updateCountdown();
+    const interval = setInterval(updateCountdown, 1000);
+
+    return () => clearInterval(interval);
+  }, [announcements]);
```

Comment:
Once we hear back from design regarding the number of announcements, we should clean this up. If design decides to only choose one announcement to display, we can just store that first announcement outside of this `useEffect` hook.